library/src/main/java/com/novoda/downloadmanager/NotificationDispatcher.java

```diff
@@ -29,7 +37,18 @@ void updateNotification(DownloadBatchStatus downloadBatchStatus) {
             NotificationInformation notificationInformation = notificationCreator.createNotification(downloadBatchStatus);
             DownloadBatchStatus.Status status = downloadBatchStatus.status();
 
-            if (status == DOWNLOADED || status == DELETION || status == ERROR) {
+            downloadService.dismissStackedNotification(notificationInformation);
+            if (downloadBatchStatus.notificationSeen()) {
+                Log.v("DownloadBatchStatus: ", downloadBatchStatus.getDownloadBatchId(), " notification has already been seen.");
```

Comment:
There is no need to add " " between params of one of the Log methods, they are added automatically when concatenating params.
Just FYI 😉

Reply:
I forgot 😄